### PR TITLE
Make Dark Mode Automatic

### LINKED ISSUE #175: [Feature] Automatic light/dark theme based on system theme
I've run up a small instance for some users and the one "complaint" that I've had is that it's light or dark theme only. Modern websites and browsers can check for the system theme - would it be possible to do similar here?

I'm a little broken atm but will try to look at this in a few months (fingers crossed)

### PR DESCRIPTION
**Pull request by Kangie**
Use `@media (prefers-color-scheme:` to  determine which CSS to load based on the browser's preference.

This change consolidates all CSS into `style.css` and removes the manual 'Dark Mode' toggle.

Also grabbed 'development' flavours of both Prism CSS themes to make them consistent and updated Prism to 1.29.

Closes #175

### CODE REVIEW

**Comment by mtlynch** [comment]
Thanks for contributing this!

Would it be possible to narrow the scope down to just what's necessary for automatic dark mode? So we'd revert the changes to `emberstack/sftp` and Prism because they're outside of what's necessary for the change.

I'd also like to keep the third-party code in the `third-party` dir, as mixing it in with first-party (LogPaste-specific) code makes it hard to do version updates in the future.

**Comment by Kangie** [comment]
It might be possible to use `@import` directives to directly include the relevant CSS; I'll have a play.

I'm going to need to make changes to the Prism code anyway; one version is minified and the other is not - I may as well retain the upgrade in a new commit.

I won't revert the emberstack change - that fixes CI breakage. As long as you don't squash commits when you merge the PR each of these should be an atomic change.

**Comment by mtlynch** [comment]
>I won't revert the emberstack change - that fixes CI breakage. As long as you don't squash commits when you merge the PR each of these should be an atomic change.

Can you pull this into a separate PR and make that change first?

Keep in mind, this PR will be squashed before merging, so it needs to merge cleanly with that in mind.

**Comment by Kangie** [comment]
Requested changes made; Prism still updated because I'd have had to dig out an older version to make both consistent.

**Comment by Kangie** [comment]
> Also grabbed 'development' flavours of both Prism CSS themes to make them consistent 

The old state was one minified and one 'development'; for consistency and ease of development / tinkering they're all 'development' flavours now, but it we need to save a few K per request maybe adding a CSS minifying step to the release-generating CI is the best solution.

It's beyond the scope of this PR regardless!

File: static/css/style.css
Comment (mtlynch) [inline static/css/style.css:3]: I'd like to stick with separate files for light and dark. Can we change to:

```css
@import url(light.css) (prefers-color-scheme: light);
@import url(../third-party/prism/default.css) (prefers-color-scheme: light);

@import url(dark.css) (prefers-color-scheme: dark);
@import url(../third-party/prism/dark.css) (prefers-color-scheme: dark);
```

Let's roll back the modifications to the Prism files, as we don't need to modify them with this method, and updating Prism is outside the scope of the PR.

**Comment by mtlynch** [comment]
>>Keep in mind, this PR will be squashed before merging,

>Why squash a PR with atomic commits? I could understand if the commits were:
>
>* Do thing
>* Fix bug in do_thing
>* make CI happy
>
>I rebased and force pushed any CI/CD catches, so squashing this PR before I split it for you seems like it's being done for opaque reasons and has has added an additional 200% overhead to a simple change.

I understand that you have your own workflow that's comfortable for you.

I maintain several open-source projects, and I do it just as a hobby, so I arrange workflows that optimize for my efficiency as the maintainer.

I appreciate and value contributions like yours, but it's a nontrivial increase in my time to adapt the PR review workflow for each individual contributor.

All my repos are exclusively squash and merge because that yields a consistent result regardless of the contributor's commit hygiene.

Force pushing actually makes the reviews harder because I can't compare the current version to the last version I reviewed, so I've now disabled them for this repo.

**Review by mtlynch — CHANGES_REQUESTED** [review]
Thanks for splitting it up! This is much easier to review.

**Review by mtlynch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
